docs/articles/documentation/continuous-integration/bitbucket-pipelines.md

```diff
@@ -0,0 +1,112 @@
+
+## Step 2 - Enable Bitbucket Pipelines for Your Project
+
+To enable Bitbucket Pipelines for your project, create a `bitbucket-pipelines.yml` file in the project's root folder.
+
+Bitbucket Pipelines CI allows you to avoid manual environment configuration and use Docker containers set up to work with Chrome and Firefox. You can [use a TestCafe Docker image](#option-1---use-testcafe-docker-image) with pre-installed software.
+However, if you already have a Docker image prepared to deploy your web application and run tests, you can [install TestCafe on this image before testing](#option-2---install-testcafe-on-a-docker-image).
+
+For more information about how to get started with Pipelines, see the [Atlassian Bitbucket support page](https://confluence.atlassian.com/bitbucket/get-started-with-bitbucket-pipelines-792298921.html).
+
+### Option 1 - Use TestCafe Docker Image
+
+Specify the `testcafe/testcafe` image name in the `image` field of the [bitbucket-pipelines.yml file](https://confluence.atlassian.com/bitbucket/configure-bitbucket-pipelines-yml-792298910.html). Then, configure a pipeline that installs the project's dependencies and triggers a custom TestCafe launcher when you push commits and create pull requests.
+
+```yaml
+image: testcafe/testcafe
+pipelines:
+  pull-requests:
+    '**':
+      - step:
+          script:
+            - npm ci
+            - /opt/testcafe/docker/testcafe-docker.sh firefox:headless,chromium tests/**/*
+
+  branches:
+    master:
+      - step:
+          script:
+            - npm ci
+            - /opt/testcafe/docker/testcafe-docker.sh firefox:headless,chromium tests/**/*
+```
+
+The custom launcher script `/opt/testcafe/docker/testcafe-docker.sh` prepares the environment to run a browser and starts TestCafe. Its arguments are standard TestCafe [command line parameters](../using-testcafe/command-line-interface.md).
+
+Commit and push this file to your repository.
+
+### Option 2 - Install TestCafe on a Docker Image
+
+Use the `image` field in the [bitbucket-pipelines.yml file](https://confluence.atlassian.com/bitbucket/configure-bitbucket-pipelines-yml-792298910.html) to specify a desired Docker image with Node.js and browsers installed. Then, configure a pipeline that installs the project's dependencies and runs tests when you push commits and create pull requests.
+
+```yaml
+# Replace '10.14' with the latest Node.js LTS version
+# available on Docker Hub
+image: circleci/node:10.14-browsers
+pipelines:
+  pull-requests:
+    '**':
+      - step:
+          script:
+            - npm ci
+            - npm test
+
+  branches:
+    master:
+      - step:
+          script:
+            - npm ci
+            - npm test
+```
+
+To specify how npm should run TestCafe, add the `test` script to the project's `package.json` file and execute the `testcafe` command in this script.
+
+The following example shows a command that runs tests in Chromium in headless mode.
```

Comment:
The following example shows a command that runs tests in Chromium in headless mode: